released/packages/coq-library-fol/coq-library-fol.1.0+8.17/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+version: "1.0+8.17"
+maintainer: "kirst@cs.uni-saarland.de"
+homepage: "https://github.com/uds-psl/coq-library-fol/"
+dev-repo: "git+https://github.com/uds-psl/coq-library-fol/"
+bug-reports: "https://github.com/uds-psl/coq-library-fol/issues"
+authors: [
+  "Dominik Kirst"
+  "Johannes Hostert"
+  "Andrej Dudenhefner"
+  "Yannick Forster"
+  "Marc Hermes"
+  "Mark Koch"
+  "Dominique Larchey-Wendling"
+  "Niklas Mück"
+  "Benjamin Peters"
+  "Gert Smolka"
+  "Dominik Wehr"
+]
+license: "CECILL-2.1"
```

Comment:
```suggestion
license: "MIT"
```
According to the `LICENSE` file in the tarball and https://github.com/uds-psl/coq-library-fol/issues/2 this should be MIT now.